lib/partitioned.rb

```diff
@@ -10,7 +9,6 @@
 require 'partitioned/partitioned_base/configurator/reader'
 require 'partitioned/partitioned_base/partition_manager'
 require 'partitioned/partitioned_base/sql_adapter'
-require 'partitioned/partitioned_base/redshift_sql_adapter'
```

Comment:
I removed Redshift support.

It used https://github.com/arp/activerecord-redshift-adapter which stopped being supported after Rails 3, so it wasn't working on Rails 4 anyway.

I didn't think it'd be important to update it to Rails 5, so I just removed it